src/constants/productTokens.ts

```diff
@@ -1,6 +1,6 @@
 import * as tokenAddresses from 'constants/ethContractAddresses'
 import bedBorderLogo from 'assets/bed-border.png'
-//import bedLogo from 'assets/bedlogo.png'
+import dataLogo from 'assets/bed-border.png'
```

Comment:
naive but I'm guessing we don't have a DATA logo yet so this is a TODO?

Reply:
correct, this is still in the works. We currently only have a temp logo